### ISSUE TITLE
[BUG] <Enter bug title>

### ISSUE THREAD

**Issue by theobarkerh** [Azure/azure-iot-explorer#492]
**Describe the bug**
Application binary not notarized for Mac OSX during build/package process.

**To Reproduce**
1. Download the .DMG file for Mac OS x64 from the Releases tab on this GitHub project
2. Double-click on the DMG file to open the installer
3. Drag the image to the application folder in the installer
4. Click on the application icon in Mac OS Launchpad
5. System displays error message box that app could not be opened "... because Apple cannot check it for malicious software. This software needs to be updated. Contact the developer for more information." 

**Expected behavior**
A notarized application from Azure GitHub account would just launch

**Screenshots**

**Desktop (please complete the following information):**
 - OS: OSX
 - OS Version: 12.1 Monterrey Intel x64
 - App version: 0.14.5

**Additional context**
This was also a problem with the Azure Storage Explorer application: https://github.com/microsoft/AzureStorageExplorer/issues/2395

**Comment by rido-min** [comment]
Hi @theobarkerh 

thanks for reporting this issue, would you mind updating the title?

**Comment by PeteDouma** [comment]
Any estimate when this will be resolved?

**Comment by YingXue** [comment]
Relates to #503 
Are you able to unblock yourself with the 'run/trust the app anyway' option?

**Comment by theobarkerh** [comment]
Yes, I am able to unblock with the run/trust... option. Should not have to do that.
